src/test/unit/socket.test.ts

```diff
@@ -0,0 +1,31 @@
+import chai from "chai";
+import { app } from "../../main/app";
+
+const server = app.listen(8080);
+const socketio = require("socket.io");
+const client = require("socket.io-client");
+
+describe("Socket io", () => {
```

Comment:
this looks very similar to the functional test, it's starting the app, so it's not really a unit test
if you can mock things maybe, otherwise functional tests have already covered this

Reply:
will remove as probably redundant